rero_ils/modules/items/jsonschemas/items/item-v0.0.1.json

```
@@ -552,6 +556,133 @@
           }
         }
       }
+    },
+    "url": {
+      "type": "string",
+      "title": "URL",
+      "placeholder": "Example: http://server.mediatheque.ch/CH-000019-X:223094.file",
+      "format": "uri",
+      "pattern": "^(ftp?|http?)://.*$",
+      "minLength": 7,
+      "form": {
+        "validation": {
+          "messages": {
+            "patternMessage": "Should be a valid URL format."
+          }
+        },
+        "templateOptions": {
+          "cssClass": "editor-title"
+        },
+        "hide": true,
+        "navigation": {
+          "essential": true
+        }
+      }
+    },
+    "pac_code": {
+      "title": "Conservation and preservation policy",
```

```suggestion
      "title": "PAC code",
```

And add a description: "Preservation and conservation policy"

---

@BadrAly Here we should add the description "Preservation and conservation policy"